plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -47,9 +47,7 @@ const getInputElementFromEvent = (
 	let inputElement = null;
 
 	if ( event.target instanceof HTMLButtonElement ) {
-		inputElement = event.target.parentElement?.querySelector(
-			'.input-text.qty.text'
-		);
+		inputElement = event.target.parentElement?.querySelector( '.qty' );
```

Comment:
This isn't needed for the fix, but given that it's a small change, I included it here. We discussed it in the [WooCommerce Slack](https://woocommercecommunity.slack.com/archives/C03MCL9RTAR/p1751984653541299), there was no reason why this selector needed to include the `.input-text` and `.text` classes, as `.qty` is more intentional than the other two. In the future, this might make it easier for extensions to reuse this logic.

Reply:
Could we do the same thing [here](https://github.com/woocommerce/woocommerce/blob/68855a96f8ba6248328fe09b4c5d66c6bb0dc42a/plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/style.scss#L38) in the styles?